doc/metadata_routing.rst

```diff
@@ -205,8 +214,6 @@ least one metadata. For instance, if an estimator supports ``sample_weight`` in
   ``my_weights`` is done at the router level, and not by the object, e.g.
   estimator, itself.
 
-Metadata are requested in the same way for scorers using ``set_score_request``.
```

Comment:
This is important and not covered in the above list, why remove it?

Reply:
Yes, but you already wrote this in the beginning of the list:

```
For instance, if an estimator supports ``sample_weight`` in ``fit`` and ``score``, it exposes 
``estimator.set_fit_request(sample_weight=value)`` and ``estimator.set_score_request(sample_weight=value)``.
```

And the list is general enough to apply to both methods.

Reply:
scorers don't have a score method, they're a callable, that's why they're different.

Reply:
Okay, I didn't know that. But it seems to me that doesn't make a difference for users.

Could you please have a look at lines 208-211:
```
 For instance, if an estimator supports
``sample_weight`` in ``fit`` and ``score``, it exposes
``estimator.set_fit_request(sample_weight=value)`` and
``estimator.set_score_request(sample_weight=value)``. 
```
I think `set_score_request` is already covered there. So if we keep the line:
`Metadata are requested in the same way for scorers using ``set_score_request``.`
then should we exclude the scoring from above?

Reply:
>  Metadata are requested in the same way for scorers using ``set_score_request``.

Yes, that statement covers it, but you have removed it here.

Reply:
Sorry, @adrinjalali, I meant to include it back yesterday, but I've just seen that I forgot to push. I didn't mean to make it difficult.